src/Client.ts

```diff
@@ -350,7 +350,7 @@ export class Client {
      *
      * @param handler  Handler function to call on transfer progress.
      */
-    trackProgress(handler: ProgressHandler) {
+    trackProgress(handler: ProgressHandler | undefined) {
```

Comment:
What about using `trackProgress(handler?: ProgressHandler)`? Then, you can also call `trackProgress()`.

Reply:
You're totally right, I'll straighten that out right away. :-)

Reply:
Yes, good catch. Maybe replace it with something like "If no handler is provided..."?